helm/dns-operator-azure/templates/policyexception.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.kyvernoPolicyExceptions.enabled }}
+{{- if .Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException" -}}
```

Comment:
How do we solve the ordering issue here? I mean, if this app is installed before `kyverno` has applied the CRDs`, the `dns-operator-azure` will be deployed without the `PolicyException`, but we always want the to `PolicyException` be created, right?

Reply:
You are right. I think we can remove the capabilities check completely. If the user wants to disable it, there is already a toggle. What do you think?

Reply:
I think we need to add PolicyException like this directly inside of kyverno-app https://github.com/giantswarm/kyverno-app/pull/291.

Reply:
I updated the PR as we talked in the slack. This operator can start after kyverno so it can deploy its own PolicyException.